packages/web-components/src/Powerbox.js

```diff
@@ -0,0 +1,21 @@
+/* global powerbox */
+
+class Powerbox {
+  constructor() {
+    this.loaded = false;
+    window.addEventListener('load', () => {
+      this.loaded = true;
+      this.expandPetdata();
+    });
+  }
+
+  expandPetdata() {
+    if (!this.loaded) return;
+    if (typeof powerbox === 'undefined') {
+      throw new Error('powerbox not found');
```

Comment:
I think throwing an error here is a bit too drastic, as it results in throws when the powerbox isn't installed.  The powerbox is not yet mandatory.

Reply:
Changed to logging a warning as discussed